package.json

```diff
@@ -35,6 +35,7 @@
     "wds": "wds --app-index dev/index.html --watch --node-resolve",
     "preserve": "yarn prep",
     "serve": "concurrently -c \"cyan,blue\" \"yarn:rollup.watch\" \"yarn:wds\" --kill-others",
+    "storybook:axe": "yarn storybook.prod && yarn axe-storybook --build-dir=storybook-static/",
```

Comment:
`storybook.axe` would follow the pattern below.

Reply:
I updated the code as suggested